### PR TITLE
Improves the docs: OCRing files in languages other than English + fixes typos

### PR DESCRIPTION
**Pull request by JaimeObregon**
Starts a "Troubleshooting" in the docs and fixes some typos.

### CODE REVIEW

File: docs/requirements.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ should work) that has the following software installed on it:
 
 * `Python3`_ (with development libraries, pip and virtualenv)
 * `GNU Privacy Guard`_
-* `Tesseract`_
+* `Tesseract`_, plus it's language files matching your document base.
```
Comment (danielquinn) [inline docs/requirements.rst]: Quick correction: this should be `its` rather than `it's`.

File: docs/troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Consumer warns ``OCR for XX failed``
+------------------------------------
+
+If you find the OCR accuracy to be too low, and/or the document consumer warns that ``OCR for
+XX failed, but we're going to stick with what we've got since FORGIVING_OCR is enabled``, then you
+might need to install the `Tesseract language files
+<http://packages.ubuntu.com/search?keywords=tesseract-ocr>`_ marching your documents languages.
+
+As an example, if your documents are written in Spanish you may need to run::
```
Comment (danielquinn) [inline docs/troubleshooting.rst]: We can't assume that our users are all using Debian or some derivitive thereof, so this should probably say something like:

> As an example, if your documents are written in Spanish and you're running a Debian-based system, you may need to run::

**Comment by danielquinn** [comment]
I've been meaning to add a troubleshooting section, so thank you for posting this!  It's all good stuff, so if you can fix just those two bits, I'll merge it right away.

**Comment by JaimeObregon** [comment]
Fixed! Note to self: do not submit pull request on Github when under sleep deprivation :-)

**Comment by danielquinn** [comment]
Thanks for this!

**Pull request merged**